src/Worker/Core/Logs.cs

```diff
@@ -9,15 +9,48 @@ namespace Microsoft.DurableTask
     /// <summary>
     /// Log messages.
     /// </summary>
-    /// <remarks>
-    /// NOTE: Trying to make logs consistent with https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/Logging/LogEvents.cs.
-    /// </remarks>
     static partial class Logs
     {
         [LoggerMessage(EventId = 15, Level = LogLevel.Error, Message = "Unhandled exception in entity operation {entityInstanceId}/{operationName}.")]
         public static partial void OperationError(this ILogger logger, Exception ex, EntityInstanceId entityInstanceId, string operationName);
 
         [LoggerMessage(EventId = 55, Level = LogLevel.Information, Message = "{instanceId}: Evaluating custom retry handler for failed '{name}' task. Attempt = {attempt}.")]
         public static partial void RetryingTask(this ILogger logger, string instanceId, string name, int attempt);
+
+        [LoggerMessage(EventId = 600, Level = LogLevel.Information, Message = "'{Name}' orchestration with ID '{InstanceId}' started.")]
+        public static partial void OrchestrationStarted(this ILogger logger, string instanceId, string name);
+
+        [LoggerMessage(EventId = 601, Level = LogLevel.Information, Message = "'{Name}' orchestration with ID '{InstanceId}' completed.")]
+        public static partial void OrchestrationCompleted(this ILogger logger, string instanceId, string name);
+
+        [LoggerMessage(EventId = 602, Level = LogLevel.Information, Message = "'{Name}' orchestration with ID '{InstanceId}' failed.")]
+        public static partial void OrchestrationFailed(this ILogger logger, Exception ex, string instanceId, string name);
+
+        [LoggerMessage(EventId = 603, Level = LogLevel.Information, Message = "'{Name}' activity of orchestration ID '{InstanceId}' started.")]
+        public static partial void ActivityStarted(this ILogger logger, string instanceId, string name);
+
+        [LoggerMessage(EventId = 604, Level = LogLevel.Information, Message = "'{Name}' activity of orchestration ID '{InstanceId}' completed.")]
+        public static partial void ActivityCompleted(this ILogger logger, string instanceId, string name);
+
+        [LoggerMessage(EventId = 605, Level = LogLevel.Information, Message = "'{Name}' activity of orchestration ID '{InstanceId}' failed.")]
+        public static partial void ActivityFailed(this ILogger logger, Exception ex, string instanceId, string name);
+
+        /// <summary>
+        /// Creates a logger named "Microsoft.DurableTask.Worker" with the specified subcategories.
+        /// </summary>
+        /// <param name="loggerFactory">The logger factory to use to create the logger.</param>
+        /// <param name="subcategories">The subcategory of the logger. For example, "Activities" or "Orchestrations".
```

Comment:
Is this `subcategories` because you can introduce more layers than just one? If so, can you include an example in the comment of multiple subcategory layers/how to delimit them? Or can we change it to subcategory?

Reply:
This is standard .NET logging stuff, so nothing here is specific to Durable. More information on categories and subcategories can be found [here](https://learn.microsoft.com/dotnet/core/extensions/logging?tabs=command-line#log-category).

Reply:
Yes, I know it's just .NET logging, my complaint is more that we named the param `subcategories` then refer to it as `subcategory` in the comment without really extrapolating anything. So, I'd recommend we either rename the param to `subcategory` or update the comment to say `subcategories` and include an example that has multiple levels.

Reply:
Renamed to `subcategory`.